lib/src/index.js

```diff
@@ -33,7 +33,6 @@ var JSONRPC = function () {
    * Initializes a JSONRPC instance..
    * @param {object} methods - The JSONRPC methods to handle.
```

Comment:
I bumped the version of babel, it appears to have cleaned up its indentation logic a little.